scalableminds/webknossos
# Use New Task Terminology in Code

### URL of deployed dev instance (used for testing):
- https://taskterminology.webknossos.xyz

### Steps to test:
- Create some tasks, request, trace finish, look at report pages, task list view, project list view, should all still look as expected

### TODOs:
- [x] adapt schema
- [x] adapt backend code
- [x] snapshots
- [x] adapt frontend code
  - [x] open_tasks_report_view
- [x] test wk
- [x] write sql evolution
- [x] adapt python client https://github.com/scalableminds/webknossos-libs/pull/930
  - [x] discuss how much backwards compatibility is needed [→none :tada: ]
- [x] test python

### Issues:
- fixes #7218

------
